about-pses/example-pse.md

```diff
@@ -29,64 +29,189 @@ Create a function named `winner` that takes two arguments, the move for `player_
 
 ## Prompts
 
-<!-- Question 1 -->
 <!-- prettier-ignore-start -->
 ### !challenge
 * type: paragraph
-* id: 164a62ea-f936-4c6a-8b74-6d8248ba9c9d
-* title: Ask Clarifying Questions
+* id: 6957fd47-9dba-48b9-ac9c-38315cda0b28
+* title: Describe Your Understanding
 * topics: pse
 ##### !question
 
-List three or more questions whose answers would clarify the problem statement. For each question, provide an answer which includes the effect your decision would have on how you would approach the problem.
+Before you begin solving this problem, take a moment to think like a professional software engineer. 
+- What do we know about the problem? 
+- What assumptions can we make based on the information in the problem statement? 
+- What further information do the example inputs and outputs give us?
+- What questions would you ask a teammate, product manager, or interviewer to better understand the problem before writing any code?
 
-##### !end-question
+<br>
+
+In the box below, list 5 or more observations about the problem or questions whose answers would clarify the problem statement. For each observation or question, include information on why that observation is important or why you are asking the question
+- For each observation, answer how that observation will affect your approach to the problem
+- For each question, describe what you are hoping to clarify about the problem and provide an answer which includes the effect your decision would have on how you might approach the problem.
+
+<br>
 
+As you come up with observations and questions, assume that error handling for invalid data is managed outside the function. We want to focus on the core behavior of the function we will write. 
```

Comment:
Repeat from comment on `highest-rated.md`

```suggestion
As you come up with observations and questions, assume that error handling for invalid data is managed outside the function. We want to focus on the core behavior of the function we will write. Examples of invalid data we don't need to manage may include:
- Mismatched data types: If a function expects a string, you may assume it will receive a string. This follows for nested data as well.
- String Sanitization: You may assume strings will not include invalid characters.
```